portal/src/components/roles-and-groups/list/common/ActionButtonCell.tsx

```diff
@@ -8,10 +8,11 @@ interface ActionButtonCellProps {
   text: string;
   onClick?: (e: any) => void;
   disabled?: boolean;
+  variant?: "destructive" | "actionButton";
```

Comment:
Name `actionButton` as `default`, because your component is already a `ActionButtonCell`.